narratives/ncov_sit-rep_2020-05-01.md

```diff
@@ -0,0 +1,1743 @@
+* [About this data](https://nextstrain.org/narratives/ncov/sit-rep/2020-05-01?n=3).
+* [Updates for Russia](https://nextstrain.org/narratives/ncov/sit-rep/2020-05-01?n=4).
+* [Updates for the Czech Republic](https://nextstrain.org/narratives/ncov/sit-rep/2020-05-01?n=7).
+* [Updates for Turkey](https://nextstrain.org/narratives/ncov/sit-rep/2020-05-01?n=8).
+* [Updates for Poland](https://nextstrain.org/narratives/ncov/sit-rep/2020-05-01?n=9).
+* [Data from Hungary, Slovakia, Slovenia and Latvia](https://nextstrain.org/narratives/ncov/sit-rep/2020-05-01?n=10).
+* [What you can do](https://nextstrain.org/narratives/ncov/sit-rep/2020-05-01?n=11).
+* [Scientific credit](https://nextstrain.org/narratives/ncov/sit-rep/2020-05-01?n=12).
+
+<!-- This is right-side text -->
+```auspiceMainDisplayMarkdown
+# Executive summary
+
+We analyzed 4,442 publicly shared COVID-19 genomes. By comparing these viral genomes to each other, we can characterize how COVID-19 is moving around the world and spreading locally. In this week's updates, we focus on Central & Eastern Europe and report:
+
+* Many introductions to Russia  
+* Heterogeneous outbreak in Moscow  
+* Majority of St. Petersburg outbreak tied to a single introduction, most likely from Western Europe  
+* Extensive mixing of cases across Turkey following repeated travel-related introductions  
+* At least 5 local transmission chains in the Czech Republic, mostly likely due to repeated introductions from Western Europe
+* At least 11 separate introductions to Poland
+* Early samples from Hungary, Latvia, Poland, Slovakia and Slovenia  
+```
+
+
+<!-- ############ SLIDE BREAK ############# -->
+
+<!-- This is left-side text 2-->
+# [COVID-19 Resources](https://nextstrain.org/ncov/europe/2020-05-01?d=tree&f_region=Europe&legend=closed)
+We've prepared some resources that will make interpreting the data we present in this narrative easier.
+#### Nextstrain Resources  
+* [START HERE: How to read a phylogeny](https://nextstrain.org/narratives/trees-background/).  
+* [Background on coronaviruses](https://nextstrain.org/help/coronavirus/human-CoV).
+* [Common misconceptions](https://nextstrain.org/narratives/ncov/sit-rep/2020-03-13?n=11).
+
+#### External Resources  
+* [How SARS-CoV-2 mutates and spreads](https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html).
+* [Ask a Scientist & FAQs](https://covid19.fas.org/).
+* [WHO Situation Reports](https://www.who.int/emergencies/diseases/novel-coronavirus-2019/situation-reports).
+* [CDC Resources](https://www.cdc.gov/coronavirus/2019-ncov/index.html).
+* [Why the coronavirus is so confusing](https://www.theatlantic.com/health/archive/2020/04/pandemic-confusing-uncertainty/610819/).
+<!-- There is no right-side text -->
+
+
+<!-- ############ SLIDE BREAK ############# -->
+
+<!-- This is left-side text 3-->
+# [A note on sampling](https://nextstrain.org/ncov/europe/2020-05-01?c=country&r=country&d=map&p=grid&legend=closed)
```

Comment:
```suggestion
# [A note on sampling](https://nextstrain.org/ncov/europe/2020-05-01?c=country&r=country&d=map&f_region=Europe&p=grid&legend=closed)
```
I think this is right, to filter to Europe? I think this will help see the Europe circles better, hopefully, if the zoom on the map goes in a bit!